rosdep/python.yaml

```
@@ -412,6 +412,19 @@ pyflakes3:
       packages: [pyflakes]
   rhel: ['python%{python3_pkgversion}-pyflakes']
   ubuntu: [pyflakes3]
+pygeodesy-pip:
```

From [CONTRIBUTING.md](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#pip-1):
> Python packages, which are only available on [PyPI](https://pypi.org/), should use the prefix python3- and suffix -pip to avoid colliding with future keys from package managers.
```suggestion
python3-pygeodesy-pip:
```

Reminder that the stanza will need to be moved once the key is changed to maintain alphabetical order.